applications/ConvectionDiffusionApplication/custom_utilities/move_particle_utility.h

```diff
@@ -1271,7 +1271,7 @@ namespace Kratos
 
 							if (keep_looking)
 							{
-								KRATOS_THROW_ERROR(std::logic_error, "FINISHED THE LIST AND COULDNT FIND A FREE CELL FOR THE NEW PARTICLE!", "");
+								KRATOS_THROW_ERROR(std::logic_error, "FINISHED THE LIST AND COULDN'T FIND A FREE CELL FOR THE NEW PARTICLE!", "");
```

Comment:
Does the apostrophe here need to cancelled out, i.e. `\'` ?

Reply:
Nope.